packages/vanilla/src/components/checkbox/checkbox.md

```diff
@@ -0,0 +1,25 @@
+# Checkbox
+
+## Purpose
+
+Common form control component that allows the user to choose one or more options. 
```

Comment:
I feel like this is a source of confusion from the start - this pattern is called checkbox, but it's describing a checklist. Maybe we should have a checklist pattern, because having a single standalone checkbox is also a legitimate use case.

Reply:
We can add a story for a single checkbox and change the default to "Checklist" to make it more clear.